package.json

```diff
@@ -257,6 +257,15 @@
       {
         "fileMatch": "apollo.config.yaml",
         "url": "./schemas/apollo.config.schema.json"
+      },
+      {
+        "fileMatch": "apollo-router.yaml",
+        "_url": "https://github.com/apollographql/router-template/raw/main/.apollo/router_config_schema.json",
+        "url": "./schemas/router_config_schema.json"
```

Comment:
I'm not sure if I should link to the schema on GitHub or a local copy here. Any thoughts @dylan-apollo?